nixos/tests/all-tests.nix

```diff
@@ -476,6 +476,7 @@ in {
   image-contents = handleTest ./image-contents.nix {};
   orangefs = handleTest ./orangefs.nix {};
   os-prober = handleTestOn ["x86_64-linux"] ./os-prober.nix {};
+  osquery = handleTestOn ["x86_64-linux"] ./osquery.nix {};
```

Comment:
aarch64-linux, aarch64-darwin, x86_64-darwin are unsupported? if so, please mark it in the package too if it's relevant.

Reply:
Let's start with `x86_64-linux`